src/masonite/commands/NewCommand.py

```diff
@@ -14,8 +14,12 @@ class NewCommand(Command):
         {target? : Path of you Masonite project}
         {--b|--branch=False : Specify which branch from the Masonite repo you would like to install}
         {--r|--release=False : Specify which version of Masonite you would like to install}
+        {--rp|--repo=False : Specify from which repository you want to craft your: girardinsamuel/masonite-base-project}
```

Comment:
it would be better he instead of doing =False just specify our cookie-cutter repo that it installs from now

Reply:
Yes I agree 😉